sig-ui/annual-report-2021.md

```diff
@@ -0,0 +1,102 @@
+
+As we have no subproject other than the Dashboard, there is no real feedback loop besides GitHub Issues and Slack channel traction.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+Slide Deck: https://docs.google.com/presentation/d/1W4NioOkAF2VFiu-5t80p2vlu3_OznpugiyiViFuitaM/edit?usp=sharing
+
+Recording: https://youtu.be/ZyUQiN3S6TE?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ&t=837
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+Yes
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+We measure activity by activity within the GitHub repo. Filing/responding to issues and submitting/reviewing PRs.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+We currently have no metric for reviewer/approver bandwidth. We've worked in a communal manner to ensure nothing becomes stale. This has generally worked over the last several years, and isn't a major concern for us.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+The SIG doesn't participate in any activities or programs explicitly meant to help grow contributors. We attempt to be responsive and welcoming within our issues and PRs to try and retain those who take the first step and engage in conversation however.
+
+**What programs do you participate in for new contributors?**
+
+We attempt to maintain `Good First Issue` labels on things that we think early-lifecycle contributors could begin with. During KubeCons we also do introduction presentations and mentoring sessions to try and funnel contributors in to our SIG. 
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+Yes, current chairs are spread between Kubermatic (formerly Loodse) and Red Hat. 
+
+## Current initiatives and project health
+
+In addition to the ongoing effort of maintenance, we are improving support visualizing various resource objects and adding the ability for installations to have custom themes.
+
+The often-unspoken reality is, a UI or Dashboard for Kubernetes is often a differentiator for many companies. Additionally, there are many different ways and opinions on what a front-end for Kubernetes should look like.
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+When the Dashboard was originally written, it used a pull/polling design. Now, we are working towards supporting the shared informer pattern, making the Dashboard more-real-time.
```

Comment:
is there a KEP or any issues set for this?

Reply:
Updated with links to the issues :)